L10nSnapshotTests/L10nMktSuiteSnapshotTests.swift

```diff
@@ -0,0 +1,164 @@
+
+        sleep(2)
+        navigator.performAction(Action.SetURLByTyping)
+        // app.staticTexts["sourdough pizza"].tap()
+        app.staticTexts.element(boundBy: 2).tap()
+        sleep(1)
+
+        userState.url = "sourdough"
+        navigator.performAction(Action.SetURLByTyping)
+        sleep(2)
+        snapshot("URLBar-02")
+    }
+    
+    // DarkMode for these tests
+    func test3DefaultTopSites() {
+        // Enable Dark Mode
+        navigator.goto(SettingsScreen)
+        navigator.goto(DisplaySettings)
+        app.switches["SystemThemeSwitchValue"].tap()
+    
+        app.cells.staticTexts.element(boundBy: 6).tap()
+        navigator.goto(HomePanelsScreen)
+        snapshot("DefaultTopSites-01")
+    }
+    
+    func test4PrivateBrowsingTabsEmptyState() {
+        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+        app.buttons["closeTabButtonTabTray"].tap()
+        waitForExistence(app.tables["Empty list"], timeout: 3)
+        snapshot("PrivateBrowsingMode")
+    }
+    
+    func test5DefaultSearchEngine() {
+        navigator.goto(SearchSettings)
+        XCTAssert(app.tables.staticTexts["Google"].exists)
+        snapshot("SearchSuggestions")
+        
+        // Disable Dark Mode
+        navigator.goto(SettingsScreen)
+        navigator.goto(DisplaySettings)
+        app.switches["SystemThemeSwitchValue"].tap()
+    }
+    
+    func testSearchWidgets2() {
+        // Set a url in the pasteboard
+        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
+        navigator.openURL("pocket.com")
+        waitUntilPageLoad()
+        navigator.performAction(Action.PinToTopSitesPAM)
+
+        // Navigate to topsites to verify that the site has been pinned
+        navigator.nowAt(BrowserTab)
+        navigator.openNewURL(urlString: "mozilla.org")
+        waitUntilPageLoad()
+        navigator.performAction(Action.PinToTopSitesPAM)
+        
+        setupSnapshot(springboard)
+        // Open the app and set it to background
+        app.activate()
+        sleep(1)
+        snapshot("DefaultBrowser-01")
+        sleep(1)
+        XCUIDevice.shared.press(.home)
+        
+        // Swipe Right to go to Widgets view
+        let window = springboard.children(matching: .window).element(boundBy: 0)
+        window.swipeRight()
+        window.swipeRight()
+        // Swipe Up to get to the Edit and Add Widget buttons
+        // This line is needed the first time widgets view is open
+        springboard.alerts.firstMatch.scrollViews.otherElements.buttons.element(boundBy: 0).tap()
+        
+        let element = springboard/*@START_MENU_TOKEN@*/.scrollViews["left-of-home-scroll-view"]/*[[".otherElements[\"Home screen icons\"].scrollViews[\"left-of-home-scroll-view\"]",".scrollViews[\"left-of-home-scroll-view\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
```

Comment:
Long line, possible to break down?

Reply:
Actually when you see it in xcode it does not look like that:
<img width="870" alt="Captura de pantalla 2021-03-17 a las 9 56 20" src="https://user-images.githubusercontent.com/1897507/111440900-157d8300-8707-11eb-89d3-9d8679a7622b.png">

I don't think we can modify this...it is not our app's element but springboard...there are no ids as far as I know that we can use.
I know this is not nice :(